code/go/0chain.net/smartcontract/storagesc/helper_test.go

```diff
@@ -171,6 +172,18 @@ func updateBlobber(t testing.TB, blob *StorageNode, value currency.Coin, now int
 	return ssc.addBlobber(tx, input, balances)
 }
 
+func updateBlobber(t testing.TB, blob *StorageNode, value currency.Coin, now int64,
```

Comment:
looks like it is duplicate of `updateBlobberAdd`

Reply:
Yes but in `updateBlobberAdd` we are calling `ssc.addBlobber(tx, input, balances)` , its used to test that we cannot perform any updates using `addBlobber` now

Reply:
@Hitenjain14 if `updateBlobberAdd` would confuse users, let's rename it to `addBlobber`.

Reply:
Its a helper test function , using `addblobber` to update the blobber which should fail and thats what we check .